dart/dynamics/MetaSkeleton.hpp

```diff
@@ -71,6 +71,11 @@ class MetaSkeleton : public common::Subject
   /// Default destructor
   virtual ~MetaSkeleton() = default;
 
+  /// Returns mutex.
+  virtual std::unique_ptr<common::MutexReference> getCustomMutex() const = 0;
```

Comment:
We could consider naming this something like `getMutexReference()` and keep the original `getMutex()` function the way it is.

Reply:
I renamed this to `getLockableReference()` instead to reflect the renaming of `MutexReference` to `LockableReference`.